apps/explorer/lib/explorer/chain.ex

```diff
@@ -5109,6 +5109,109 @@ defmodule Explorer.Chain do
     end
   end
 
+  def fetch_latest_delegate_call_txn(address_hash) do
+    query = from(
+      txn in Transaction,
+      where: txn.to_address_hash == ^address_hash or txn.created_contract_address_hash == ^address_hash,
+      where: txn.status == ^1,
+      order_by: [desc: txn.block_number],
+      limit: 10
+    )
+    results = Repo.all(query)
+
+
+    fetch_internal_transaction_for_delegate_call(results)
+  end
+
+  def fetch_internal_transaction_for_delegate_call(list) do
```

Comment:
this method will fetch internal transaction on rpc to check whether this transaction include a delegatecall, if found, return latest delegatecall "to" address, if not return nil